Update index.md to make content more up to date (#3471)
the-turing-way/the-turing-way: book/website/index.md

(welcome)=
# Welcome

_Welcome to The Turing Way Handbook to Reproducible, Ethical, and Collaborative Data Science._

The Turing Way is an open science, open collaboration, and community-driven project. 
We involve and collaborate with a diverse community of contributors to make data science accessible, comprehensible and beneficial for everyone.

```{admonition} Top Tip
:class: tip

> _The Turing Way_ is not meant to be read from start to finish. 
> Start with what interests you!
> Whether it's a concept, a tool, or a method you need right now, jump in.
> Browse the different guides or use the search box —
> it's all about what you want to learn.

Author: ```suggestion
_The Turing Way_ is not meant to be read from start to finish. 
The Turing Way is not meant to be read from start to finish. Start with a concept, tool or method that you need now, in your current work. Browse the different guides that make up the book, or use the search box to search for whatever you would like to learn about first.
```

> it's all about what you want to learn.

Author: How do I reject this and go back to what was there?

Reviewer: You could suggest an edit which removes all of these lines. Or, if this was a single commit you could revert that commit.

(welcome)=
# Welcome

_Welcome to The Turing Way Handbook to Reproducible, Ethical, and Collaborative Data Science._

> _The Turing Way_ is an open source book, a global collaboration and a supportive community.

Reviewer 2: I'm kind of sad to lose "community-driven" in this edit - what's the rationale behind that change @malvikasharan?

Author: Me too. I am sorry that I have not been following this thread until now. Let me do the review.

Author: ```suggestion
_The Turing Way_ is an open science, open collaboration, and community-driven project. 
```

We involve and collaborate with a diverse community of contributors to make data science accessible, comprehensible and beneficial for everyone.

```{admonition} Top Tip
:class: tip

> _The Turing Way_ is not meant to be read from start to finish. Start with what interests you!

Reviewer 2: ```suggestion
_The Turing Way_ is not meant to be read from start to finish. 
Start with what interests you!
```

Whether it's a concept, a tool, or a method you need right now, jump in.
Browse the different guides or use the search box —
it's all about what you want to learn.
```

Our moonshot goal is to make reproducibility "too easy not to do"! 
Whether you are a researcher, coder, project leader, community connector, funder, policymaker, or just curious about data, your unique role is crucial in this journey.

Let's make data science inclusive, together! 

> Visit our [start page](https://the-turing-way.start.page/) to get started. 🚀

Reviewer 2: Directing people to the start page feels a little circular - the first link on the start page links back to this page. 
The start page is great for a links summary for people to get more involved in the project - maybe we should introduce it with that context?

Author: ```suggestion
```
This shouldn't be here as we do have a paragraph below.

```{figure} figures/theturingway-pathway.*
---
name: welcome-image
alt: The Turing Way project is illustrated as a road or path with shops for different data science skills. People can go in and out with their shopping cart and pick and choose what they need.
---
_The Turing Way_ guides - an illustration by Scriberia and _The Turing Way_ community shared under the CC-BY 4.0 license on Zenodo. [DOI: 10.5281/zenodo.3332807](http://doi.org/10.5281/zenodo.3332807)
```

## Different Pathways

(welcome-community)=
## Join _The Turing Way_

Our goal is to provide individuals and organizations in academia, industry, government, and the public sectors the information they need to ensure that their projects are easy to reproduce and reuse. 
We encourage all stakeholders in data science to utilise _The Turing Way_ to understand their roles and responsibilities for reproducibility.

**What Awaits You**

* Collaboration Beyond Boundaries: Engage in a global collaboration that transcends borders and welcomes diverse perspectives.

> * Skill Development: Learn a skills, share a skill that you have and contribute to projects in making a real impact.

Reviewer 2: ```suggestion
* Skill Development: Learn or share a skill that you have and contribute to projects in making a real impact.
```
There's a typo here but I'm not sure what it should be corrected to.

Author: I would like to go back to the original content: https://book.the-turing-way.org/#our-community - it's simple! What do you think?

> * Learn, Remix, Reuse: Dive into our resources, and feel free to remix and reuse materials (just remember to attribute as per our [License](https://github.com/the-turing-way/the-turing-way/blob/main/LICENSE.md)].

Reviewer 2: ```suggestion
* Learn, Remix, Reuse: Dive into our resources, and feel free to remix and reuse materials (just remember to attribute as per our [License](https://github.com/the-turing-way/the-turing-way/blob/main/LICENSE.md)).
```

**Calling Everyone In: Your Participation and Perspectives Matter!**

The Turing Way community is a open and collaborative space. 
It is an ecosystem where all stakeholders of data science come together to make research "too easy not to do." 
We invest in socio-technical skills, fostering a team spirit to build something greater than any individual effort.

```{admonition} Translation
:class: tip
Call for translators! We’re looking for contributors to help translate this space for everyone! Read more about our translation workflow and how you can contribute to it in the {ref}`translation chapter<ch-translation>`.
```

```{figure} figures/theturingway-chapters.*
---

> name: theturingway-chapters

Reviewer 3: OK, so this is an actual issue as the same figure with the same name got actually merged into the book at a separate page: https://github.com/the-turing-way/the-turing-way/blob/main/book/website/foreword/navigate.md and now there is a clash with the references.

@the-turing-way/infrastructure-working-group - do we have a preferred way to deal with the reuse of figures at multiple locations? The easy workaround is to use a different name, but I wonder how that works with the the figure gallery/index page (or shall we cross that bridge when we finally get there?). 
Also, is there a way to reuse the alt-text and have the exact same thing for both?

Reviewer: I'm not sure if there is a way to have the same figure in two places.

If you put the Markdown for the figure in a separate file and included it in two places you would still get the figure name clash.

It will look odd in the figure gallery to have two identical figures though.

Reviewer 4: Ping @agoose77?

Reviewer 2: I think we should just use a different name.
I haven't been keeping up with [the gallery discussions](https://github.com/the-turing-way/the-turing-way/issues/2046) but I'd sort of been assuming that that would load images from the figures folder (or wherever [we end up keeping images in the future](https://github.com/the-turing-way/the-turing-way/issues/3843)) rather than having a script go through the book and see what figures are actually used.

Reviewer 3: For the gallery, I would definitely would love to see both a "Gallery of everything" as a couple of the really cool illustrations are not necessarily made for TTW but for adjacent projects. Maybe this could live in a separate repo.

And I would like to see a Table of Figures, for everything we use in the book. And here all the duplicates should hopefully be found and cleaned up/made self-consistent.

Reviewer 3: (That said, in this PR I agree that we can be pragmatic and use a different name, and to figure out these gallery and figure policies separately.)

Reviewer 5: @sgibson91 oops. Missed the `mention` part here.

@JimMadge in JB2 (`mystmd`), you can embed figures such that the embedded figures do not increment reference counts:

````markdown
:::{figure} image.png
:name: img-1

An image!
:::

See @img-1, or look below:

![](#img-1)

An image appears!
````

In the EB stack, we don't have a way to duplicate a figure without creating effectively a new figure.

Reviewer: That's great! Another thing to look forward to in MystMD :smile:

> name: theturingway-chapters

Reviewer 2: ```suggestion
name: theturingway-chapters2
```
